changelog/5025.feature.rst

```diff
@@ -0,0 +1,2 @@
+Added the ability to dynamically scale `sunpy.visualization.animator` instances.
+By specifying a ``clip_interval``, it will now clip the minimum and maximum at each slider step to the specified interval.
```

Comment:
```suggestion
By specifying the ``clip_interval`` keyword, it will now clip the minimum and maximum at each slider step to the specified interval.
```